index.rb

```diff
@@ -1,6 +1,6 @@
 require "rubygems"
 require "sinatra"
-require "datamapper"
+require "data_mapper"
```

Comment:
It's actually using `datamapper` :neutral_face:

Reply:
for some reason I couldn't get it to run even after installing the deps unless I changed it like this, you wouldn't happen to have an idea why? (It basically said there was no module "datamapper" and once I renamed it then it seemed happy)

Reply:
I have backed out that change for now, since it can still be done separately if it turns out to be necessary once I understand better why datamapper didn't like me :scream:

Reply:
It could be a Ruby version incompatibility, given the age of the library.

Reply:
The ruby version shipped in the docker container does seem to be 2.5.0 based on this error output:

```
site_1  | /var/www/fll.presidentbeef.com/index.rb:3:in `require': cannot load such file -- datamapper (LoadError)
site_1  | 	from /var/www/fll.presidentbeef.com/index.rb:3:in `<top (required)>'
site_1  | 	from config.ru:4:in `require'
site_1  | 	from config.ru:4:in `block in <main>'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/rack-2.0.6/lib/rack/builder.rb:55:in `instance_eval'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/rack-2.0.6/lib/rack/builder.rb:55:in `initialize'
site_1  | 	from config.ru:1:in `new'
site_1  | 	from config.ru:1:in `<main>'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/lib/unicorn.rb:56:in `eval'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/lib/unicorn.rb:56:in `block in builder'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/lib/unicorn/http_server.rb:795:in `build_app!'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/lib/unicorn/http_server.rb:660:in `init_worker_process'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/lib/unicorn/http_server.rb:682:in `worker_loop'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/lib/unicorn/http_server.rb:549:in `spawn_missing_workers'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/lib/unicorn/http_server.rb:563:in `maintain_worker_count'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/lib/unicorn/http_server.rb:293:in `join'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/gems/unicorn-5.4.1/bin/unicorn:126:in `<top (required)>'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/bin/unicorn:23:in `load'
site_1  | 	from /var/www/fll.presidentbeef.com/vendor/ruby/2.5.0/bin/unicorn:23:in `<main>'
```

Reply:
@presidentbeef I just added a dynamic begin/rescue that tries datamapper first, and on LoadError will try data_mapper next. Do you think that might work as a permanent solution?

See here: https://github.com/presidentbeef/fll-site/pull/1/files#diff-ae2011d4228e01c2ca87dab5b839be4bR3